develop-docs/self-hosted/troubleshooting.mdx

```diff
@@ -23,6 +23,18 @@ CSRF_TRUSTED_ORIGINS = ["https://sentry.example.com", "http://10.100.10.10", "ht
 
 See [Django's documentation on CSRF](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-CSRF_TRUSTED_ORIGINS) for further detail.
 
+## Snuba
+
+### No event showing on the dashboard
```

Comment:
I don't think we should advertise this actually, since rust-consumer is used in production that should be fixed.